### DISCUSSION TITLE
Urgent: Frustration with SMS 2FA Country Availability {Morocco}

### DISCUSSION THREAD

**Discussion by Vvoox** [community/community#126074] [Actions]
### Urgent: Frustration with SMS 2FA Country Availability

2FA Feedback

Dear GitHub Support Team,

I hope this message finds you well. I am writing to express my deep frustration and disappointment regarding the mandatory implementation of two-factor authentication (2FA) on GitHub. While I understand the importance of securing my account, the process has been nothing short of a nightmare due to a critical issue: my country is not listed in the available options for receiving SMS verification codes.

This omission is incredibly inconvenient and has left me feeling alienated as a user. SMS is my preferred method for 2FA due to its simplicity and accessibility, and the lack of support for my country is unacceptable. It feels as though GitHub is disregarding the needs of users from certain regions, which is both frustrating and disheartening.

The inability to use SMS 2FA is causing significant disruption to my workflow, and I am now seriously considering moving my projects and collaborations to an alternative platform that better accommodates my security preferences and geographical needs.

I urge you to address this issue promptly and expand the list of supported countries for SMS 2FA. If no solution or workaround is provided, I will have no choice but to leave GitHub and seek a more inclusive and user-friendly platform.

Thank you for your immediate attention to this matter. I look forward to a swift resolution.

Sincerely,
KHALIL DAOULAT 

<img width="1241" alt="Screenshot 2024-05-27 at 21 04 21" src="https://github.com/community/community/assets/48126333/cbf6475f-4ffd-4351-8c3c-de3d79577ddc">

**Comment by leofun01** [comment]
I have the same problem. Here I collected info about other CI-servers for migration:
- [codeberg.org](https://codeberg.org/)
- [sourceforge.net](https://sourceforge.net/)
- [sr.ht](https://git.sr.ht/)
- [notabug.org](https://notabug.org/)
- [gitea.com](https://gitea.com/)
- [gitlab.com](https://about.gitlab.com/)
- [disroot.org](https://git.disroot.org/)
- [savannah.gnu.org](https://savannah.gnu.org/)

Also you can create [new ticket](https://support.github.com/contact/account?contact[type]=account) with the issue.
